CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please add your changelog entry under this comment in the correct category (Security, Fixed, Added, Changed, Deprecated, Removed - in this order).
 -->
 
+### Changed
+* Remove deprecated set-output in docker-scan job
```

Comment:
Maybe use "Replace" / "Fix" and prefix with `(internal)`.

Reply:
Fine with me, should i remove it in a new PR or do we wait until the next changes to remove it?